nipype/interfaces/workbench/cifti.py

```diff
@@ -154,3 +154,278 @@ class CiftiSmooth(WBCommand):
     input_spec = CiftiSmoothInputSpec
     output_spec = CiftiSmoothOutputSpec
     _cmd = "wb_command -cifti-smoothing"
+
+
+class CiftiCorrelationInputSpec(CommandLineInputSpec):
+    in_file = File(
+        exists=True,
+        mandatory=True,
+        argstr="%s ",
+        position=0,
+        desc="The input ptseries or dense series",
+    )
+    out_file = File(
+        name_source=["in_file"],
+        name_template="correlation_matrix_%s.nii",
+        keep_extension=True,
+        argstr=" %s",
+        position=1,
+        desc="The output CIFTI",
+    )
+
+    roi_override = traits.Bool(
+        exists=True,
+        argstr="-roi-override %s ",
+        position=2,
+        desc=" perform correlation from a subset of rows to all rows",
+    )
+
+    left_roi = File(
+        exists=True,
+        position=3,
+        argstr="-left-roi %s",
+        desc="Specify the left roi metric  to use",
+    )
+
+    right_roi = File(
+        exists=True,
+        position=5,
+        argstr="-right-roi %s",
+        desc="Specify the right  roi metric  to use",
+    )
+    cerebellum_roi = File(
+        exists=True,
+        position=6,
+        argstr="-cerebellum-roi %s",
+        desc="specify the cerebellum meytric to use",
+    )
+
+    vol_roi = File(
+        exists=True,
+        position=7,
+        argstr="-vol-roi %s",
+        desc="volume roi to use",
+    )
+
+    cifti_roi = File(
+        exists=True,
+        position=8,
+        argstr="-cifti-roi %s",
+        desc="cifti roi to use",
+    )
+    weights_file = File(
+        exists=True,
+        position=9,
+        argstr="-weights %s",
+        desc="specify the cerebellum surface  metricto use",
+    )
+
+    fisher_ztrans = traits.Bool(
+        position=10,
+        argstr="-fisher-z",
+        desc=" fisherz transfrom",
+    )
+    no_demean = traits.Bool(
+        position=11,
+        argstr="-fisher-z",
+        desc=" fisherz transfrom",
+    )
+    compute_covariance = traits.Bool(
+        position=12,
+        argstr="-covariance ",
+        desc=" compute covariance instead of correlation",
+    )
+
+
+class CiftiCorrelationOutputSpec(TraitedSpec):
+    out_file = File(exists=True, desc="output CIFTI file")
+
+
+class CiftiCorrelation(WBCommand):
+    r"""
+    Compute correlation from CIFTI file
+    The input cifti file must have a brain models mapping on the chosen
+    dimension, columns for .ptseries or .dtseries,  
+    >>> cifticorr = CiftiCorrelation()
+    >>> cifticorr.inputs.in_file = 'sub-01XX_task-rest.ptseries.nii'
```

Comment:
This file needs to exist (but should be empty) in https://github.com/nipy/nipype/tree/master/nipype/testing/data to allow the doctests to work.